eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -8,6 +8,9 @@ parameters:
   # Branch of the VMR to use (to push to for internal builds)
   vmrBranch: $(Build.SourceBranch)
 
+  # True when the build is a full build (not a PR build or lite build)
```

Comment:
When the ci.yml calls into this template it is going to run all the legs.  Is that wasting compute given the ci-lite.yml is running on each commit?  Should this parameter be renamed/refactored to behave as an isExtendedBuild?  With this the each leg would be considered extended or not.  It would not be possible to run all legs with one pipeline.

Reply:
> Is that wasting compute given the ci-lite.yml is running on each commit?

I agree on this. The decision on splitting the legs so that the full pipeline is an extension of the lite pipeline rather than a full run of all the legs comes down to a preference on usability. I can see it being frustrating to have one of the legs in the extended pipeline fail, make a change to address the failure, and then having to repeat runs on both pipelines in order to confirm that the change does not affect legs in the lite pipeline.

Reply:
I see, then from that perspective, would it make sense that we only utilize ci-lite in all PRs and only for pushes to main?  We shouldn't run it on pushes to release branches if we are going to always run the ci pipeline on every push as well.

Reply:
I think that makes sense. Lite would be run for all PRs and pushes to main, and the ci pipeline would run on every push to branches except for main, with main running on a schedule.

Reply:
Update the pipeline triggers in https://github.com/dotnet/installer/pull/17370/commits/a0f1e49155c565fc1052096bdbeec7b0ccd9873b